utils/debug/registerCache.go

```diff
@@ -135,17 +135,5 @@ func (c *fileRegisterCache) Persist() error {
 	if err != nil {
 		return err
 	}
-	err = f.Sync()
-	if err != nil {
-		return err
```

Comment:
should we keep one of the syncs here ?

Reply:
bufio.Writer does require calling `Flush`, everything else does that automagically on Close.